tests/test_isPrimaryFlag.py

```diff
@@ -36,31 +176,120 @@ def setUp(self):
         # Load sample input from disk
         expPath = os.path.join(getPackageDir("pipe_tasks"), "tests", "data", "v695833-e0-c000-a00.sci.fits")
         self.exposure = afwImage.ExposureF(expPath)
+
+        # Characterize the image (create PSF, etc.)
+        charImConfig = CharacterizeImageConfig()
+        charImTask = CharacterizeImageTask(config=charImConfig)
+        self.charImResults = charImTask.run(self.exposure)
+
         # set log level so that warnings do not display
         Log.getLogger("calibrate").setLevel(Log.ERROR)
 
     def tearDown(self):
         del self.exposure
 
-    def testIsPrimaryFlag(self):
+    def testIsSinglePrimaryFlag(self):
         """Tests detect_isPrimary column gets added when run, and that sources
         labelled as detect_isPrimary are not sky sources and have no children.
         """
-        charImConfig = CharacterizeImageConfig()
-        charImTask = CharacterizeImageTask(config=charImConfig)
-        charImResults = charImTask.run(self.exposure)
         calibConfig = CalibrateConfig()
         calibConfig.doAstrometry = False
         calibConfig.doPhotoCal = False
         calibTask = CalibrateTask(config=calibConfig)
-        calibResults = calibTask.run(charImResults.exposure)
+        calibResults = calibTask.run(self.charImResults.exposure)
         outputCat = calibResults.outputCat
         self.assertTrue("detect_isPrimary" in outputCat.schema.getNames())
         # make sure all sky sources are flagged as not primary
         self.assertEqual(sum((outputCat["detect_isPrimary"]) & (outputCat["sky_source"])), 0)
         # make sure all parent sources are flagged as not primary
         self.assertEqual(sum((outputCat["detect_isPrimary"]) & (outputCat["deblend_nChild"] > 0)), 0)
 
+    def testIsScarletPrimaryFlag(self):
+        """Test detect_isPrimary column when scarlet is executed as the deblender
```

Comment:
Hmmm, it passed flake8... But you are right.

Reply:
So, it looks like the `setup.cfg` in `pipe_tasks` does not yet have the `max-doc-length = 79` condition, so that's why it "passed".  I won't insist, but it wouldn't hurt to run a manual flake8 on the files you've touched it just to save any similar violations needing fixing when the condition is added, e.g.
```
flake8 --ignore=E133,E226,E228,N802,N803,N806,N812,N813,N815,N816,W503 --max-line-length=110 --max-doc-length=79 tests/test_isPrimaryFlag.py
```